packages/app/src/app/components/ContextMenu/index.tsx

```diff
@@ -20,7 +26,10 @@ type Item = ItemType | ItemType[];
 type ChildrenProps =
   | {
       childFunction: true;
-      children: (onContextMenu: OnContextMenu) => React.ReactNode;
+      children: (
+        onContextMenu: OnContextMenu,
+        disclosureProps: any
```

Comment:
I think using `unknown` is a better choice.

```suggestion
        disclosureProps: unknown
```